src/bundle.rs

```diff
@@ -159,6 +176,16 @@ impl Bundle {
         }
         true
     }
+
+    pub fn write_to_disk_human_readable(
+        &self,
+        mut output: Box<dyn Write + Send>,
+    ) -> Result<(), std::io::Error> {
+        let content = format!("{self}");
+        output.write_all(content.as_bytes())?;
```

Comment:
suggestion: would `write!(&mut output, "{}", self)?; work here? That may avoid creating the intermediate string.